### PR TITLE
Faq

### LINKED ISSUE #254: [PE-D][Tester B] Incomplete FAQ in User Guide
**Description**
The FAQ section in the User Guide (UG) lacks coverage of some fundamental aspects of the application, such as what happens when a user inputs an invalid command.

**Steps to Reproduce:**
1. Open the MediContact User Guide.
2. Navigate to the FAQ section.

**Expected Result:**
The FAQ section should provide answers to basic questions, including what happens if the user inputs an invalid command.

**Actual Result:**
The FAQ section does not include any details regarding invalid command handling, which is crucial for novice users to understand how the system reacts to incorrect usage.

**Proposed Solution:**
Add more questions

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: TechRaj/ped#19

### PR DESCRIPTION
**Pull request by LynetteLeeSiYing**
**Main Changes**
1. Edit Quick Start
2. Include target audience in introduction
3. Update FAQ

close #254

### CODE REVIEW

**Review by OttoTan — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -755,7 +760,23 @@ Furthermore, certain edits can cause the AddressBook to behave in unexpected way
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous MediContact home folder.
+
+**Q**: What should I do if I input an invalid command?<br>
+**A**: Check out [Features](#features) or simple run `help` to see the full list of commands, and try entering a valid command again.
```
Comment (myrilla12) [inline docs/UserGuide.md:766]: Small typo, can consider replacing 'simple' with 'simply'

File: docs/UserGuide.md [hunk 2]
```diff
@@ -755,7 +760,23 @@ Furthermore, certain edits can cause the AddressBook to behave in unexpected way
...
+
+**Q**: Where should I start if I am a novice to CLI?<br>
+**A**: Run `help` to check the format. For more information on each command's usage, check out the [Features](#features) section.
+
+**Q**: Can I have duplicate contacts in my address book? <br>
+**A**: Duplicate names are not permitted. Contacts with the same name, even if capitalized differently, are considered duplicates.
+
+**Q**: Can I mark certain patients as high-priority?<br>
```
Comment (myrilla12) [inline docs/UserGuide.md:774]: Can consider rewording this question to marking patients as 'favourite' or 'more important` instead. High-priority could potentially imply that there's a feature allowing them to set patient contacts as low, medium or high priority.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -772,23 +793,23 @@ Furthermore, certain edits can cause the AddressBook to behave in unexpected way
...
-| **Import** | `import FILENAME.json`<br>e.g. `import patientRecords.json`  |
-| **List**   | `list` <br/>`list *` (to list starred contacts)              |
-| **Note**   | `note NAME [ap/APPOINTMENT] [m/MEDICATION] [r/REMARK]…`<br> e.g.,`note John Doe r/Allergic to XXX m/10mg Ibuprofen` |
-| **Sort**   | `sort` (to sort contacts based on appointment dates)         |
-| **Star**   | `star INDEX` or `star NAME` <br/> e.g., `star 3`, `star Alex Yeoh` |
-| **Unstar** | `unstar INDEX` or `unstar NAME` <br/> e.g., `unstar 3`, `unstar Alex Yeoh` |
-| **View**   | `view INDEX` or `view NAME` <br/> e.g., `view 3`, `view Alex Yeoh` |
+| **Clear**  | `clear`                                                                                                                                                                                                                             |
+| **Delete** | `delete INDEX` or `delete NAME` <br> e.g., `delete 3`, `delete Alex Yeoh`                                                                                                                                                           |
```
Comment (myrilla12) [inline docs/UserGuide.md:800]: Can consider removing the `.` after the `e.g` in this table. Seems unnecessary.

**Review by myrilla12 — COMMENTED** [review]
Looks good! Just a few small suggestions.

File: docs/images/Ui.png
Comment (Bonzzz3) [inline docs/images/Ui.png:1]: Ui seems a bit too small for me and I think it should show at least 2/3 full contacts. Also the bottom part is cut off. Perhaps you can state what is the problem with the current Ui and I can change it later.
image

**Review by Bonzzz3 — CHANGES_REQUESTED** [review]
LGTM, just that the Ui doesn't adhere to requirement. Can just delete the Ui and lmk which part of the current Ui you would like me to change.

**Review by ritvi12 — APPROVED** [review]
LGTM!

**Review by Bonzzz3 — APPROVED** [review]

**Pull request merged**